src/class-plugin-curated-posts.php

```diff
@@ -71,6 +72,7 @@ public function with_query_context( array $context, array $attributes, WP_Block_
 		if ( is_string( $search_term ) && strlen( $search_term ) > 0 ) {
 			$args['s'] = $search_term;
 		}
+		$args['post__not_in'] = [ get_the_id() ]; // phpcs:ignore WordPressVIPMinimum.Performance.WPQueryParams.PostNotIn_post__not_in
```

Comment:
What if I **want** to the current post to be available?

Reply:
@renatonascalves I guess I could add a checkbox to allow it or something like that.

Reply:
maybe I should revert that part of the change and make a new issue to handle that. I really want to get this bug fix out.

Reply:
Got it! Feel free to move ahead. I was just picturing this would introduce changes that might affect other use cases.